modules/mlops/kubeflow-platform/policies/iam-profile-sa-policy.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "iam:GetRole",
+                "iam:UpdateAssumeRolePolicy"
```

Comment:
This is too wide and needs to get permissive on a role name pattern

Reply:
Change added to PR